index.html

```diff
@@ -15,7 +15,7 @@
 </head>
 
 <body>
-    <nav class="navigation spacing flex-row items-center wrap py-2 px-6 fixed left-0 top-0 z-2 w-100">
+    <nav class="navigation spacing flex-row items-center wrap py-2 px-6 fixed left-0 top-0 z-10 w-100">

```

Comment:
what is z property?

Reply:
z-index, I followed the tailwind naming convention for that.

Reply:
awesome